docs/cli/interactive.md

```
@@ -0,0 +1,383 @@
+create    import    list    multisig    ledger    biometric    recovery    backup    restore
+
+galaxy> wallet create --<Tab>
+--name    --network
+```
+
+### Command History
+
+Navigate through your command history using arrow keys:
+
+| Key | Action |
+|-----|--------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)